tests/src/api_surface_fwd.cpp

```
@@ -0,0 +1,63 @@
+// other moth_ui header. Forward-declared types are used in function signatures
+// to confirm every declaration in the file is parseable.
+
+#include "moth_ui/moth_ui_fwd.h"
+
+#include <catch2/catch_all.hpp>
+
+// Use each forward-declared class in a function signature. These functions are
+// never called; the compile step is what matters. Grouping them in an anonymous
+// namespace suppresses unused-function warnings.
+namespace {
+    void pin_graphics(moth_ui::IRenderer*, moth_ui::IImage*, moth_ui::IFont*,
```

Clang (14.0.6)

[error] all parameters should be named in a function

(readability-named-parameter,-warnings-as-errors)